liquidhaskell-boot/src/Language/Haskell/Liquid/Bare.hs

```diff
@@ -703,15 +703,20 @@ makeSpecRefl cfg src menv specs env name sig tycEnv = do
   where
     lawMethods   = F.notracepp "Law Methods" $ concatMap Ghc.classMethods (fst <$> Bare.meCLaws menv)
     mySpec       = M.lookupDefault mempty name specs
-    rflSyms      = S.fromList (getReflects specs)
+    -- Collect reflected symbols and fully qualify them
+    rflSyms      = S.fromList $ qualifySym . localize <$> (getReflects specs)
+    localize :: F.Symbol -> F.LocSymbol
+    localize sym = maybe (dummyLoc sym) Bare.varLocSym $ L.lookup sym (Bare.reSyms env)
```

Comment:
The purpose of localize is escaping me at the moment.

Reply:
Addressed in a2bd4dd698e671a1b041909d20bc133cd73089c6